hoot-services/Makefile

```diff
@@ -62,6 +62,7 @@ clean-db-force:
 	rm -rf ./src/main/java/hoot/services/db2
 
 test: config
+	echo Running services tests...
```

Comment:
Yeah, I guess tests are a special case. They all give some succinct status. I think this is fine.